docs/docs-source/docs/modules/run-cloudflow-gcp-marketplace/pages/run-sample-application.adoc

```diff
@@ -0,0 +1,102 @@
+= Run a Sample Cloudflow Application on GCP
+:toc:
+:toc-title: ON THIS PAGE
+:toclevels: 2
+
+== Setting up the Cloudflow `kubectl` Plugin
+
+With Cloudflow installed on a GKE cluster through the Marketplace, you can interact with Cloudflow via its `kubectl` plugin: `kubectl cloudflow`.
+If you haven't already, download and set up the `kubectl` plugin following the https://cloudflow.io/docs/current/get-started/prepare-development-environment.html#_download_and_install_the_cloudflow_cli[instructions].
+
+Then connect your local `kubectl` to the GKE cluster on which Cloudflow is installed. You can find the command to do that from the "Clusters" view:
+
+.Connect `kubectl` to GKE cluster
+image::connect-clusters.png[Connect kubectl to GKE cluster]
+
+It should look like the following:
+[source,bash]
+----
+$ gcloud container clusters get-credentials <cluster-name> --zone <zone> --project <gcp-project>
+----
+Where cluster-name refers to your Kubernetes cluster, as identified https://console.cloud.google.com/kubernetes/list[here], and gcp-project refers to your https://cloud.google.com/resource-manager/docs/creating-managing-projects[Google Cloud project].
+
+== Building and Deploying call-record-aggregator
```

Comment:
Inform the user they have to login into docker repo before they run sbt buildApp.

Reply:
If they use GCR (which would make sense sinc ethey are on GCP), tell them to login with `gcloud auth login`